src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -50,6 +50,36 @@ internal PublicClientApplicationBuilder WithUserTokenLegacyCachePersistenceForTe
             return this;
         }
 
+        /// <summary>
+        /// Configures the public client application to use the recommended reply URI for the platform. 
+        /// See https://aka.ms/msal-net-default-reply-uri.
+        /// <list type="table">
+        /// <listheader>
+        /// <term>Platform</term>
+        /// <term>Default Reply URI</term>
+        /// </listheader>
+        /// <item>
+        /// <term>.NET desktop</term>
+        /// <term><c>https://login.microsoftonline.com/common/oauth2/nativeclient</c></term>
+        /// </item>
+        /// <item>
+        /// <term>UWP</term>
+        /// <term>value of <c>WebAuthenticationBroker.GetCurrentApplicationCallbackUri()</c></term>
+        /// </item>
+        /// <item>
+        /// <term>.NET Core</term>
+        /// <term><c>https://localhost</c></term>
```

Comment:
It's http, not https. Needs an E2E test :)

Reply:
my bad. and yes, it needs en E2E test.

Reply:
This is easy to E2E test on desktop and net core but we dont have a way of testing this on UWP

Reply:
Will be address in a later PR